SoomlaAndroidProfile/src/com/soomla/profile/events/gameservices/GetLeaderboardsFailedEvent.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2012-2015 Soomla Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.soomla.profile.events.gameservices;
+
+import com.soomla.profile.domain.IProvider;
+
+public class GetLeaderboardsFailedEvent extends BaseGameServicesEvent {
+    public final String ErrorDescription;
+
+    public GetLeaderboardsFailedEvent(IProvider.Provider provider, String errorDescription, String payload) {
+        super(provider, payload);
+        this.ErrorDescription = errorDescription;
```

Comment:
@EugeneButusov, be careful `ErrorDescription` breaks naming conventions in Java. Soomla does that in their classes, but I'm not sure you should do the same. Follow case by case.